src/pages/Results.jsx

```diff
@@ -30,15 +33,44 @@ const Results = () => {
       const width = pdf.internal.pageSize.getWidth();
       const height = pdf.internal.pageSize.getHeight();
       pdf.addImage(imgData, "JPEG", 0, 0, width, height);
-      pdf.save("download.pdf");
+      pdf.save("Result.pdf");
     })
   }
 
+
+  const [imageData, setImageData] = useState(null);
```

Comment:
This is never used? i see the effect setting the data, but then its not being referenced anywhere